css/css-grid/grid-definition/support/testing-utils.js

```diff
@@ -12,14 +12,14 @@ var TestingUtils = (function() {
         assert_in_array(getComputedStyle(element).gridTemplateRows, value, "gridTemplateRows");
     }
 
-    function testGridTemplateColumnsRows(gridId, columnsStyle, rowsStyle, columnsComputedValue, rowsComputedValue) {
+    function testGridTemplateColumnsRows(gridId, columnsStyle, rowsStyle, columnsComputedValue, rowsComputedValue, label) {
         test(function() {
             var grid = document.getElementById(gridId);
             grid.style.gridTemplateColumns = columnsStyle;
             grid.style.gridTemplateRows = rowsStyle;
             checkGridTemplateColumns(grid, columnsComputedValue);
             checkGridTemplateRows(grid, rowsComputedValue);
-        }, "'" + gridId + "' with: grid-template-columns: " + columnsStyle  + "; and grid-template-rows: " + rowsStyle + ";");
+        }, (label||'') + " '" + gridId + "' with: grid-template-columns: " + columnsStyle  + "; and grid-template-rows: " + rowsStyle + ";");
```

Comment:
Nit: if `label` is not provided, this is adding a leading space.
Nit: add spaces around `||`.
Nit: inconsistent quotes, sometimes `'` is a string delimiter and sometimes a literal character.